app/assets/javascripts/initializers/initScrolling.js.erb

```diff
@@ -199,6 +199,8 @@ function algoliaPaginate(tag){
   .then(function searchDone(content) {
     nextPage += 1;
     insertArticles(content.hits);
+    var checkBlockedContentEvent = new CustomEvent('checkBlockedContent')
```

Comment:
You can use `const` here even though it's in older JS code. The minimum browser list supported supports `const`.